daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/HexBinaryLengthParsers.scala

```diff
@@ -36,6 +36,9 @@ sealed abstract class HexBinaryLengthParser(override val context: ElementRuntime
     val nBits = getLengthInBits(start).toInt
     if (nBits == 0) {
       currentElement.setDataValue(zeroLengthArray)
+    } else if (nBits < 0) {
+      // Integer overflow occurred, recommend using blob instead of hexBinary
+      PE(start, "Data is too large for xs:hexBinary. Consider using blobs instead: https://cwiki.apache.org/confluence/display/DAFFODIL/Proposal%3A+Binary+Large+Objects")
```

Comment:
I'm a little hesitant to include this link, only because it's very possible this link might change in the future.

Reply:
I understand the hesitancy, but I figured that most people wouldn't be familiar with blobs and thought that the link might be the best way to point them in the right direction.

Reply:
If you want to include a link, we should create an Apache short link that can be kept up to date. 
I just created: https://s.apache.org/daffodil-blob-feature for this if we want to keep the link.

Reply:
Agreed, this would be a better link.

Reply:
Done